### PR TITLE
Style request form

### PR DESCRIPTION
**Pull request by hbillings**
Todo:
- [ ] Custom form field for city/state/zip
screen shot 2017-11-03 at 10 04 52 pm

- [ ] Figure out how to style `input` of `type=file`:
screen shot 2017-11-03 at 10 05 17 pm

- [ ] Custom form field for fee waiver amount
screen shot 2017-11-03 at 10 04 17 pm

- [x] Add submission and confirmation section
- [ ] Default button style is what?
- [x] Extend sidebar to footer
- [x] Top border on footer
- [x] Set fixed width on sidebar
- [x] Figure out how to get the step numbers right ("submission" is the sixth step, but it's hardcoded in our template so the code renders "step n of 5").
- [x] Restyle progress bar

### CODE REVIEW

**Comment by hbillings** [comment]
@adborden, this is ready for review!

File: js/components/foia_request_form.jsx [hunk 1]
```diff
@@ -55,13 +57,24 @@ function FoiaRequestForm({ formData, isSubmitting, onSubmit, requestForm, submis
...
       <div id="foia-request-form_submit" className="foia-request-form_submit">
-        <p>Please review the information you’ve entered and submit.</p>
-        <p>
-          You should hear from the agency you’ve submitted a records request to
-          within 10 days. If you don’t hear from the agency, please reach out
-          using the contact information provided to you on this site.
-        </p>
-        <button type="submit">Submit</button>
+        <div className="foia-request-form_inline-progress">
+          Step 6 of 6
```
Comment (adborden) [inline js/components/foia_request_form.jsx:61]: We could make this dynamic based on the sections

```
const totalSteps = requestForm.sections.length + 1;
//..
`Step ${totalSteps} of ${totalSteps}`

File: www.foia.gov/_sass/layouts/_sidebar.scss [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
-      &.tab_active a {
-        padding: 1rem 2.8rem; // magic number to keep the tab from changing size when the text is bold
+        margin-bottom: 0;
+        a {
+          padding: 1rem $space-4x;
+          display: inline-block;
+          width: 100%;
+        }
+        &.tab_active a {
+          padding: 1rem 2.8rem; // magic number to keep the tab from changing size when the text is bold
```
Comment (adborden) [inline www.foia.gov/_sass/layouts/_sidebar.scss:26]: hope no one is using custom fonts in their browser!

File: www.foia.gov/_sass/modules/_foia_request_form.scss [hunk 3]
```diff
@@ -25,19 +25,36 @@
...
   // Total hack, if we're following a field-object it is probably a section.
   // Each section already has it's own return to top link, so this is probably
   // a duplicate link at the very bottom of the form.
   .field-object + & {
     display: none;
   }
 }
+
+.foia-request-form_is-required {
+  float: right;
```
Comment (adborden) [inline www.foia.gov/_sass/modules/_foia_request_form.scss:55]: What is preventing a long label from running into this float?
screenshot from 2017-11-06 11-59-43

**Review by adborden — APPROVED** [review]
Noice!

File: www.foia.gov/_sass/layouts/_sidebar.scss (hunk 2 above)
Comment (hbillings) [reply www.foia.gov/_sass/layouts/_sidebar.scss:26]: We're killing the bold (yay!) so this rule will go away too, just like it deserves.

**Comment by adborden** [comment]
We decided this gap is needed to distinguish the sidebar from the footer?

screenshot from 2017-11-06 12-03-58

**Comment by adborden** [comment]
This isn't supposed to include any mobile styling, correct?

File: www.foia.gov/_sass/modules/_foia_request_form.scss (hunk 3 above)
Comment (hbillings) [reply www.foia.gov/_sass/modules/_foia_request_form.scss:55]: I could throw a margin on the label to keep that space open. hmm.

**Comment by hbillings** [comment]
@adborden The gap is a stand-in until I can review this with Aviva (I'm sure she'll have a more elegant solution). And nope, no mobile styling yet!

**Pull request merged**